tests/generic_config_updater/test_lo_interface.py

```diff
@@ -0,0 +1,174 @@
+import logging
+import pytest
+
+from tests.common.helpers.assertions import pytest_assert
+from tests.common.config_reload import config_reload
+from tests.generic_config_updater.gu_utils import apply_patch, expect_op_success, expect_op_failure
+from tests.generic_config_updater.gu_utils import generate_tmpfile, delete_tmpfile
+
+pytestmark = [
+    pytest.mark.topology('any'),
+]
+
+logger = logging.getLogger(__name__)
+
+@pytest.fixture(scope="module", autouse=True)
+def setup_env(duthosts, rand_one_dut_hostname, cfg_facts):
+    """
+    Setup/teardown fixture for loopback interface config
+    Args:
+        duthosts: list of DUTs.
+        rand_selected_dut: The fixture returns a randomly selected DuT.
+        cfg_facts: config facts for selected DUT
+    """
+    duthost = duthosts[rand_one_dut_hostname]
+
+    config_tmpfile = generate_tmpfile(duthost)
+    logger.info("config_tmpfile {} Backing up config_db.json".format(config_tmpfile))
+    duthost.shell("sudo cp /etc/sonic/config_db.json {}".format(config_tmpfile))
+
+    # Cleanup LOOPBACK_INTERFACE config
+    lo_interfaces = cfg_facts.get('LOOPBACK_INTERFACE', {})
+    for lo_interface in lo_interfaces:
+        del_loopback_interface = duthost.shell("sudo config loopback del {}".format(lo_interface),
+            module_ignore_errors=True)
+        pytest_assert(not del_loopback_interface['rc'],
+            "Loopback interface '{}' is not deleted successfully".format(lo_interface))
+
+    yield
```

Comment:
>yield [](http://example.com/codeflow?start=4&length=5)

Is it true that you only test on empty pre-condition? Normally the apply patch is used to replace an old ip interface.